compass/ocean/tests/drying_slope/namelist.forward

```diff
@@ -0,0 +1,28 @@
+config_run_duration='0000_12:00:01'
+config_dt='0000_00:00:30'
```

Comment:
The legacy case uses `config_dt='0000_00:00:20'` for the 250m resolution. With the current 30s dt, there are some large errors on the right boundary for the 250m case:
![ssh_depth_section_30s](https://user-images.githubusercontent.com/3514023/163596709-cc95464c-cafc-4f31-8768-ea5b213d0168.png)

For a 20 second dt:
![ssh_depth_section_20s](https://user-images.githubusercontent.com/3514023/163597092-22f0cd14-b91a-4c7d-83bb-0a2f32b66ca3.png)


I also ran it with a 10 second dt:
![ssh_depth_section_10s](https://user-images.githubusercontent.com/3514023/163596428-471c7dac-2b86-467d-b3bc-ef1167668d0a.png)

The smaller dt progressively improves the boundary mismatch and also improves the drying front (especially for r=0.0025). Maybe we should with with 10s (or even 5 since there's still a slight mismatch with 10?)

Reply:
@sbrus89 Good idea. How about 8s since that is roughly 1/4 the dt of the lower resolution case?

Reply:
Sounds good to me.